testing/gogs/APKBUILD

```diff
@@ -1,5 +1,5 @@
 # Contributor: Carlo Landmeter <clandmeter@gmail.com>
-# Maintainer:
+# Maintainer: 7heo <7heo@mail.com>
 pkgname=gogs
 pkgver=0.6.1
 pkgrel=4
```

Comment:
There's a difference between failing a package build and not building any package.